.vscode/workspace.code-workspace

```diff
@@ -60,6 +60,10 @@
             "name": "web-api",
             "path": "../packages/web-api"
         },
+        {
+            "name": "workers",
```

Comment:
can you be more specific? worker sounds too generic.

Reply:
function-workers may be?

Reply:
rename to function-apps